src/app/zap-templates/templates/app/cluster-objects-src.zapt

```diff
@@ -261,5 +261,32 @@ bool CommandIsFabricScoped(ClusterId aCluster, CommandId aCommand)
     return false;
 }
 
+bool CommandIsLarge(ClusterId aCluster, CommandId aCommand)
+{
+    switch (aCluster)
+    {
+    {{#zcl_clusters}}
+      {{#zcl_commands}}
+      {{#first}}
+      case Clusters::{{asUpperCamelCase parent.name}}::Id:
+      {
+          switch (aCommand) {
+      {{/first}}
+          {{#if isLargeMessage}}
```

Comment:
See, _here_ it's named the right thing...

Reply:
Changed to CommandHasLargePayload.
Earlier, I tried to be consistent with what was in zap, i.e., isLargeMessage. But CommandIsLargeMessage does not make sense, so kept it as CommandIsLarge. Don't care either way. Both seem ok to me.